copier.yml

```diff
@@ -26,12 +26,38 @@ schedule_interval:
   help: Cron expression
   default: 0 12 * * *
 
+use_ingestion:
+    type: bool
+    help: Would you like to use ingestion framework?
+    default: false
+
+ingestion_framework:
+    type: str
+    when: "{% if use_ingestion %}true{% endif %}"
+    help: What ingestion framework would you like to use?
+    choices:
+        Airbyte: airbyte
+    default: airbyte
+
+destination_id:
```

Comment:
In case of destination, source and workspace we cannot have one id for prod / dev 

Solution:
- create questions for dev / prod 
- leave it and use for now only dev instance

Reply:
Do you mean the id is generated automatically and that's why we can not have the same? Even if we somehow migrate the state from one Airbyte to another?
I that's true we will need to use something different than ID, is there any human readable / business id? For example destination name?

Reply:
There is an open discussion here: https://github.com/airbytehq/airbyte/issues/14448 . Neither do they recommend nor plan to enforce unique names for Airbyte resources. I am not sure if using _name_ attribute is the solution here, regardless of the convenience.

Reply:
Do we assume that we will have only one source/destination/connection or many in the project?

If many, parametrizing them in copier is a big overhead for analytics to know them while initializing the project.
The better solution in my opinion would be to create cli for retrieving existing sources / destinations / connections like it's in octavia.

If we want only one for demo purposes leaving them with id I think good enough because later this gonna be changed for upper solution